tabular/src/autogluon/tabular/models/tabular_nn/tabular_nn_model.py

```diff
@@ -674,8 +670,9 @@ def load(cls, path: str, reset_paths=True, verbose=True):
         return model
 
     def hyperparameter_tune(self, X_train, y_train, X_val, y_val, scheduler_options, **kwargs):
-        time_start = time.time()
         """ Performs HPO and sets self.params to best hyperparameter values """
+        time_start = time.time()
+        force_forkserver()
```

Comment:
I noticed that there is no ` from autogluon.core.utils.multiprocessing_utils import force_forkserver `

Reply:
fixed.